### PR TITLE
move http calls to separate class

### PR DESCRIPTION
**Pull request by splix**
Unirest doesn't work in some environments, so I added a way to use custom executor for HTTP calls. Also it's helpful for testing

### CODE REVIEW

File: src/main/java/de/raysha/lib/telegram/bot/api/RequestExecutor.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.io.File;
+import java.util.Map;
+
+/**
+ * Executes requests to Telegram Bot API
+ */
+public interface RequestExecutor {
+
+    String get(String action, Map<String, Object> parameters) throws BotException;
```
Comment (rainu) [inline src/main/java/de/raysha/lib/telegram/bot/api/RequestExecutor.java:21]: Can you pleas add some javadoc for this interface-method?

File: src/main/java/de/raysha/lib/telegram/bot/api/RequestExecutor.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    String post(String action, Map<String, Object> parameters) throws BotException;
```
Comment (rainu) [inline src/main/java/de/raysha/lib/telegram/bot/api/RequestExecutor.java:31]: Can you pleas add some javadoc for this interface-method?

File: src/main/java/de/raysha/lib/telegram/bot/api/RequestExecutor.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    String post(String action, Map<String, Object> parameters, String fileName, File file) throws BotException;
```
Comment (rainu) [inline src/main/java/de/raysha/lib/telegram/bot/api/RequestExecutor.java:43]: Can you pleas add some javadoc for this interface-method?

**Comment by rainu** [comment]
Hey. Thank you for your support. For me it looks good. A litle bit of javadoc would be nice ;) After this i will merge your pull request :)

**Comment by splix** [comment]
Sure, will do that tomorrow. Thank you

**Comment by splix** [comment]
Added a javadoc

**Pull request merged**